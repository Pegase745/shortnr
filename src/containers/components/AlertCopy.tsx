import * as React from 'react';
import { Message } from 'semantic-ui-react';

const AlertCopy: React.FC<{}> = () => (
  <Message
    success={true}
    content="Your shortened URL was copied to your clipboard!"
  />
);

export default React.memo<{}>(AlertCopy);
