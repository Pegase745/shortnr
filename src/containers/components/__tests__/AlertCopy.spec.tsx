import { shallow } from 'enzyme';
import * as React from 'react';

import AlertCopy from '../AlertCopy';

const defaultProps = {};

describe('AlertCopy.tsx', () => {
  it('should render', () => {
    const app = shallow(<AlertCopy {...defaultProps} />);

    expect(app).toMatchSnapshot();
  });
});
