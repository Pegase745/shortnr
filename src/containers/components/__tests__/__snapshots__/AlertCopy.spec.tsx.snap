// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertCopy.tsx should render 1`] = `
<Message
  content="Your shortened URL was copied to your clipboard!"
  success={true}
/>
`;
